kb/plane_geometry/formal_statements/statements/statements_scs/stat_acute_angled_triangle_height_be_inside.scs

```diff
@@ -0,0 +1,14 @@
+stat_acute_angled_triangle_height_be_inside <- statement;;
+
+stat_acute_angled_triangle_height_be_inside -> rrel_key_sc_element: concept_acute_angled_triangle; nrel_height; nrel_be_inside;;
+
+stat_acute_angled_triangle_height_be_inside => nrel_main_idtf: [Акс.(остроугольный треугольник;высота*;быть внутри*)] (*<- lang_ru;;*);;
+stat_acute_angled_triangle_height_be_inside => nrel_main_idtf: [Ax.(acute angled triangle;height*;be inside*)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]